File: app/src/main/java/com/tmiyamon/bundler/MainActivity.java
package com.tmiyamon.bundler;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onSubmit(View view) {
//        startActivity(BundlerSampleParams.apply(new Intent(this, SampleActivity.class), "name", 1));
    }
}

File: app-kotlin/src/main/java/com/tmiyamon/bundler/MainActivity.kt
package com.tmiyamon.bundler

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun onSubmit(view: View) {
        startActivity(BundlerSampleActivityParams.apply(Intent(this, SampleActivity::class.java), "name", 1));
    }
}
